### PR TITLE
[CI] Update dev_utilities

### PR DESCRIPTION
**Pull request by eliyahu77**

### CODE REVIEW

File: automation/system_test/dev_utilities.py [hunk 1]
```diff
@@ -274,10 +274,18 @@ def uninstall(redis, kafka, mysql, redisinsight):
         if redisinsight:
             cmd = "kubectl delete ingress -n devtools redisinsight"
             subprocess.run(cmd.split(), check=True)
+    except Exception as e:  # !!!
+        pass
```
Comment (liranbg) [inline automation/system_test/dev_utilities.py:278]: no need to pass if you print the exception.
also, why the `# !!!`?

File: automation/system_test/dev_utilities.py [hunk 2]
```diff
@@ -274,10 +274,18 @@ def uninstall(redis, kafka, mysql, redisinsight):
...
+        print(e)
```
Comment (Tankilevitch) [inline automation/system_test/dev_utilities.py:279]:
```suggestion
        print(e)
```

File: automation/system_test/dev_utilities.py [hunk 3]
```diff
@@ -274,10 +274,18 @@ def uninstall(redis, kafka, mysql, redisinsight):
...
+    try:
         for service in services:
             if locals().get(service):
                 cmd = f"helm uninstall {service} --namespace {namespace}"
                 subprocess.run(cmd.split(), check=True)
+    except Exception as e:  # !!!
+        print(e)
+        pass
```
Comment (Tankilevitch) [inline automation/system_test/dev_utilities.py:287]:
```suggestion
    except Exception as e:
        print(e)
```

**Review by Tankilevitch — CHANGES_REQUESTED** [review]

File: automation/system_test/dev_utilities.py [hunk 4]
```diff
@@ -274,10 +274,16 @@ def uninstall(redis, kafka, mysql, redisinsight):
         if redisinsight:
             cmd = "kubectl delete ingress -n devtools redisinsight"
             subprocess.run(cmd.split(), check=True)
+    except Exception as e:  # !!!
```
Comment (Tankilevitch) [inline automation/system_test/dev_utilities.py:277]:
```suggestion
    except Exception as e:
```

**Pull request merged**